CMakeLists.txt

```diff
@@ -352,16 +351,18 @@ else()
 endif()
 
 # testing
-copy_files_from_filelist_to_buildtree(${CMAKE_SOURCE_DIR}/cmake/installdata/test-reference-data.txt NO_INSTALL)
-add_executable(${APP_NAME_LC}-test EXCLUDE_FROM_ALL ${CMAKE_SOURCE_DIR}/xbmc/test/xbmc-test.cpp ${test_sources})
-set_target_properties(${APP_NAME_LC}-test PROPERTIES ENABLE_EXPORTS ON)
-whole_archive(_TEST_LIBRARIES ${core_DEPENDS} gtest)
-target_link_libraries(${APP_NAME_LC}-test PRIVATE ${SYSTEM_LDFLAGS} ${_TEST_LIBRARIES} lib${APP_NAME_LC} ${DEPLIBS} ${CMAKE_DL_LIBS})
-unset(_TEST_LIBRARIES)
-add_dependencies(${APP_NAME_LC}-test ${APP_NAME_LC}-libraries export-files)
-
-# Enable unit-test related targets
-if(CORE_HOST_IS_TARGET)
+if(CORE_HOST_IS_TARGET AND ENABLE_TESTING)
+  find_package(Gtest)
+  copy_files_from_filelist_to_buildtree(${CMAKE_SOURCE_DIR}/cmake/installdata/test-reference-data.txt NO_INSTALL)
+  add_executable(${APP_NAME_LC}-test EXCLUDE_FROM_ALL ${CMAKE_SOURCE_DIR}/xbmc/test/xbmc-test.cpp ${test_sources})
+  set_target_properties( PROPERTIES ENABLE_EXPORTS ON)
```

Comment:
extraneous space. also this block could use some newlines whitespace

Reply:
done, thanks!